src/Atoms/Card.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const CardWrapper = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
+const Card = (props) => {
+  // eslint-disable-next-line react/prop-types
+  return <CardWrapper>{props.children}</CardWrapper>;
```

Comment:
is there a reason to silence eslint here?

Reply:
> is there a reason to silence eslint here?

fixed